File: Python/1012.py
v1, v2, v3 = input().split()

v1 = float(v1)
v2 = float(v2)
v3 = float(v3)

triangulo = (v1 * v3)/2
circulo = 3.14159 * pow(v3, 2)
trapezio = ((v1 + v2) * v3)/2
quadrado = pow(v2, 2)
retangulo = v1 * v2

print('TRIANGULO: {:.3f}' .format(triangulo))
print('CIRCULO: {:.3f}' .format(circulo))
print('TRAPEZIO: {:.3f}' .format(trapezio))
print('QUADRADO: {:.3f}' .format(quadrado))
print('RETANGULO: {:.3f}' .format(retangulo))

File: C/1012.c
#include <stdio.h>

int main() {
 double A, B , C , tri, circ, trap, praca, retangulo;

 scanf("%lf %lf %lf ", &A, &B, &C);

  tri = ( A * C)/2;
  circ = (C * C)* 3.14159;
  trap = ((A + B)*C)/2;
  praca = B*B;
  retangulo = A*B;

 printf("TRIANGULO: %.3f\n", tri);
 printf("CIRCULO: %.3f\n", circ);
 printf("TRAPEZIO: %.3f\n", trap);
 printf("QUADRADO: %.3f\n", praca);
 printf("RETANGULO: %.3f\n", retangulo);

 
    return 0;
}